### PR TITLE
chore: update deps

### PR DESCRIPTION
**Pull request by atian25**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -4,9 +4,9 @@ module.exports = app => {
   // put before other core middlewares
   app.config.coreMiddlewares.unshift('cors');
 
-  // if security plugin enabled, and no origin config provided, only allow safe domains support CORS.
+  // if security plugin enabled, and not origin config provided, only allow safe domains support CORS.
```
Comment (dead-horse) [inline app.js]: > not origin config provided

还不如改之前呢，或者改成 origin config is not present/provided

File: package.json [hunk 2]
```diff
@@ -20,32 +20,32 @@
     "egg-plugin"
   ],
   "dependencies": {
-    "kcors": "^1.3.2"
+    "kcors": "^2.2.1"
```
Comment (dead-horse) [inline package.json]: 2.2.1 是 koa@2 的

Comment (atian25) [reply package.json]: 这种兼容的也先不升级？那我切回去。

File: app.js (hunk 1 above)
Comment (atian25) [reply app.js]: fixed

File: package.json (hunk 2 above)
Comment (atian25) [reply package.json]: 再看看

**Review by popomore — APPROVED** [review]

**Comment by atian25** [comment]
1.2.0

**Pull request merged**